### DISCUSSION TITLE
LED device disabling issue

### DISCUSSION THREAD

**Discussion by GameKnight90000** [awawa-dev/HyperHDR#921] [Q&A]
Hello! I am currently setting up my first-ever instance of HyperHDR 20.0.0.0 on a Raspberry Pi 3 model B+, using a ws2815 LED strip. I am not well-versed in RPi or LEDs, so I apologize if this is an easy fix. After installing the software using the SD image installation instructions on HyperHDR.eu, I am running into this error message, which I do not know how to resolve:

Device 'ws281x' is disabled due to an error: 'Failed to open. Error message: mmap() failed. YOUR NEED TO RUN HYPERHDR AS A ROOT USER FOR MMAP TO WORK. SUCH CONFIGURATION IS NOT OFFICIALLY SUPPORTED.'

I have searched high and low for a solution and cannot find one that solves whatever issue I have, so I appreciate any help you can provide.

**Comment by awawa-dev** [comment]
Hi
The answer is here (how to switch the service to root): https://github.com/awawa-dev/HyperHDR/wiki/FAQ
